common.blocks/page/page.bemhtml

```diff
@@ -1,28 +1,30 @@
 block('page')(
 
-    def()(function() {
+    def().match(function() { return !this._pageInit; })(function() {
         var ctx = this.ctx;
-        applyCtx([
-            ctx.doctype || '<!DOCTYPE html>',
-            {
-                tag : 'html',
-                cls : 'ua_js_no',
-                content : [
-                    {
-                        elem : 'head',
-                        content : [
-                            { tag : 'meta', attrs : { charset : 'utf-8' } },
-                            { tag : 'title', content : ctx.title },
-                            { block : 'ua' },
-                            ctx.head,
-                            ctx.styles,
-                            ctx.favicon? { elem : 'favicon', url : ctx.favicon } : ''
-                        ]
-                    },
-                    ctx
-                ]
-            }
-        ]);
+        return local({ _pageInit : true })(function() {
```

Comment:
@veged: moved it back to `local` because of https://github.com/bem/bem-xjst/issues/50 . Throws error with current bem-xjst.